de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/impl/HULoader.java

```diff
@@ -64,6 +64,11 @@
 
 public class HULoader
 {
+	public static final HULoader of(final IAllocationSource source, final IAllocationDestination destination)
```

Comment:
agree. The best would be to refactor & deprecate.
btw, are u taking care of this or shall i?